engine/sim/sim.cpp

```diff
@@ -2255,13 +2255,8 @@ void sim_t::init_fight_style()
     case FIGHT_STYLE_DUNGEON_SLICE:
       //Based on the Hero Dungeon setup
       max_time = timespan_t::from_seconds( 360.0 );
-      //Disables all raidbuffs, except those provided by scrolls or the character itself.
-      optimal_raid = 0;
-      overrides.arcane_intellect = 1;
-      overrides.battle_shout = 1;
-      overrides.power_word_fortitude = 1;
-      overrides.bloodlust = 1;
-      overrides.windfury_totem = 0;
+
+      optimal_raid = 1;
```

Comment:
Why change this to 1? Will this not again enable **all** raid buffs?

Reply:
It will have them on by default, but you can toggle them off. I guess maybe for Dslice we should turn bloodlust on and everything else off?

Before they were not toggleable, but I also thought having nothing on by default would be worse than having everything.